examples/tutorial/lessons/lesson-05-actions.adoc

```diff
@@ -0,0 +1,348 @@
+
+== Lesson 05 - Actions
+
+In the previous lessons we've learned how to define a trigger, have it fire an alert, and then how to move that alert through its life-cycle to resolution.  But in most cases we need to perform some actions in response to an alert.  Some examples:
+
+* Notify an administrator
+** e-mail, irc
+* Integrate with external system
+** snmp, aerogear, pager-duty
+* Execute a webhook
+
+Hawkular Alerting provides all of the above (and more) as action plugins.  It is also possible to create custom action plugins.
+
+Actions are tied to life-cycle.  Most actions are performed when an alert is created and moved to OPEN.  But hAlerting also allows actions to be performed on any life-cycle state transistion.  Revisiting the diagram from lesson-04, we can perform actions on any of the arrow transistions:
+
+image::image-alert-lifecycle.png[Alert Life-cycle]
+
+=== E-Mail Action Example
+
+In this example we will define a trigger with e-mail actions to be executed when we OPEN a new alert and also when it moves to RESOLVED.  We will use different e-mail addresses for each. First we define the actions and then we assign and configure the actions for use by our trigger.
+
+==== SMTP Server and Timezone
+
+Before anything else let's establish an SMTP server.  You can configure an existing SMTP server if one is available. Otherwise with this tutorial we use FakeSMTP to handle the outgoing e-mail.
+
+1) http://nilhcem.com/FakeSMTP/[Download and install FakeSMTP] as directed.
+
+2) Edit `lessons/config.properties` and set `mail.smtp.host=<your host>`.  If for some reason port 2525 is in use you can also enter a different port.
```

Comment:
No, this is correct. The stuff under tutorial/docker is for building the docker container (I will add a readme if I didn't already.   So, the config.properties in that location is a default placeholder.  The lessons/config.properties is used to override the default one at runtime.  If you are saying lessons/config.proprties is not there then that is a mistake.  I will check it..